src/main/java/org/mskcc/cbio/portal/scripts/ImportTabDelimData.java

```diff
@@ -332,6 +393,60 @@ public void importData(int numLines) throws IOException, DaoException {
         }
     }
 
+    private void expandRemainingGeneticEntityTabDelimitedRowsWithBlankValue() {
+        if (updateMode) {
+            // Expand remaining genetic entity id rows that were not mentioned in the file
```

Comment:
and what if the file contains genetic entities that were not in the `geneticAlterationMap`? Will we go back and update all other records in a similar way? 
I'm not sure this is needed at all...let's discuss.

Reply:
update: this function is adding empty columns to existing genetic_alteration records that contain a gene that was not part of the new data file (e.g. an expression file where a specific gene was not present while this gene is present in the DB because it was there in previous samples of this genetic profile).